platforms-whitepaper/v1alpha1/paper.md

```diff
@@ -81,76 +81,77 @@ identity access, infrastructure operations, and app lifecycle.
+of their organizations. At its very basic level, a platform could simply be a place where 
+users go for basic actions like user management, technical docs, or incident management if that 
+is what is most needed.
+ 
+However, just like any product, a platform can evolve over time to meet more complex needs of an organization. 
+For example, some example use cases an enterprise can meet with a platform include the following:
 
 1. Developers of products or services can automatically request a complete 
    development environment to support iterative research and development of new 
-   features. This includes spaces in relevant services such as task runners and 
-   artifact storage, membership in designated teams, and publication of connection 
+   features. This includes spaces in relevant services such as Kubernetes clusters, 
+   task runners and artifact storage, membership in designated teams, and publication of connection 
    info such as URLs and secrets.
-1. Developers of products or services can use scenario-specific code and 
-   configuration templates to rapidly bootstrap, develop and deliver new products 
-   and features
-1. Stakeholders in a product or service can observe system and user behavior in 
-   those products and services via instrumentation, dashboards and alerts
-
-A platform is bespoke to an organisation, supporting a unique set of users and 
-business needs. While application developement and delivery is often the first 
-capability a platform supports, it is important to focus on the individual needs of 
-the organization which often surfaces additional opportunities to optimise through 
-a platform offering. Some examples can include data operations, user management, and 
-incident management.
+1. Engineering Teams and SRE Teams have a central location for observability where infrastructure
+	and services can be onboarded into a single observability platform. 
+	This allows for a consistent approach to observability while giving developers 
+	an easy interface to be able to do this for any app or cloud service.
+1. Operators of third-party products and services can automatically provision 
+   spaces and supporting services to deploy and use those third-party products and 
+   services
+   
+Therefore, It is recommended that platform engineering teams start with the thinnest viable platform (TVP) [[6]], 
+which is very similar to the concept of an MVP - solve
+for the key needs of your organization with the most basic and continue to evolve your platform just like 
+any other product. 
 
 By offering consistent experiences for individual and/or scenario-oriented sets 
 of capabilities, platforms make it easy for their users to deliver valuable products.
 
 [1]: https://www.atlassian.com/devops/frameworks/team-topologies
 [2]: https://martinfowler.com/articles/talk-about-platforms.html
+[3]: https://thenewstack.io/platform-as-a-product-true-devops/
```

Comment:
I'm not yet sure how we'll do footnotes or if we want to highlight specific vendors and posts. In the meantime can we add these to the list at the end of the doc?